.github/pull_request_template.md

```diff
@@ -1,6 +1,7 @@
 # Checklist
 
-- [ ] Assign labels to PR
+- [ ] Assign labels to PR.
+- [ ] Ensure PR title matches the conventional commits format.
```

Comment:
**Suggestion:** Consider adding a checklist item to ensure that the PR description is meaningful and provides enough context for the reviewers.
```suggestion
- [ ] Assign labels to PR.
- [ ] Ensure PR title matches the conventional commits format.
- [ ] Ensure PR description is meaningful and provides enough context.
```